cmd/cephcsi.go

```diff
@@ -166,6 +168,13 @@ func main() {
 		}
 	}
 
+	// read and formulate conf.Domains from passed in domain labels
+	// TODO: decide if this will be from an init container and move as a separate image or instance
```

Comment:
Whats the advantage we are getting it if its part of init Container ?

Reply:
Using an init container keeps the ceph-csi code independent of k8s calls, and retains the CO agnostic nature of the same.